src/structs/pdb.rs

```diff
@@ -1051,6 +1051,26 @@ impl<'a> PDB {
         }
         chains
     }
+
+    /// Returns a vector of unique residue names present in the PDB file.
+    ///
+    /// # Arguments
+    ///
+    /// * `self` - A reference to the PDB struct.
+    ///
+    /// # Returns
+    ///
+    /// * `Vec<String>` - A vector of unique residue names.
+    pub fn unique_residue_names(&self) -> Vec<String> {
+        let mut resnames = Vec::new();
+        for residue in self.residues() {
+            let resname = residue.name().unwrap().to_owned();
```

Comment:
I'm assuming this unrwap here is ok because the error would be handled before, is that the case?

Reply:
> The ID or name of the Residue, it will only give a value if there is only one conformer or if all conformers have the same name
This is what the docs tell. That reminds me that the 'residue name' you get here is actually stored in the 'conformers' the additional layer added in pdbtbx to handle to full complexity of PDBs better. So it makes more sense to use the `conformers` function on the pdb, which always have a defined name. But the maybe `unique_conformer_names` makes more sense as the name for this function. The downside of this name however is that most users will not be able to tell immediately what is does, as conformer is not a widely used concept.

Reply:
Yeah, but I think conformers indeed make more sense here